packages/endpoints-organizations-registrar/test/organizations-full-controller.test.js

```diff
@@ -5835,7 +5733,7 @@ const expectedOrganization = (did, profile, services = [], overrides = {}) => {
     didNotCustodied,
     ids: idsMatcher({ did, services, includeMongoId: true }),
     normalizedProfileName: expect.any(String),
-    services: expectedServices(services, overrides.invitation),
+    services: expectedServices(services, overrides),
```

Comment:
I'm not sure the changes to expectedXXX were necessary, but since you did them you need to remove invitation from the omit list

Reply:
mostly reverted back to how you had it, [325a486](https://github.com/velocitycareerlabs/velocitycore/pull/34/commits/325a4864ee628885eb1646efdd50d544794fede4)